cmd/common.go

```diff
@@ -139,6 +143,40 @@ func listRoleTemplateBindings(ctx *cli.Context, b []RoleTemplateBinding) error {
 	return writer.Err()
 }
 
+// getCurrentServer returns the name of the name and url of the current server in the local config
+func getCurrentServer(ctx *cli.Context) (string, string, error) {
+	cf, err := loadConfig(ctx)
+	if err != nil {
+		return "", "", err
+	}
+
+	serverName := cf.CurrentServer
+	return cf.CurrentServer, cf.Servers[serverName].URL, nil
+}
+
+func getCurrentKubeConfigForUser(ctx *cli.Context, user string) (*api.Config, error) {
+	cf, err := loadConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	kubeConfig, _ := cf.FocusedServer().KubeConfigs[fmt.Sprintf(kubeConfigKeyFormat, user, cf.FocusedServer().FocusedCluster())]
+	return kubeConfig, nil
+}
+
+func setKubeConfigForUser(ctx *cli.Context, user string, kubeConfig *api.Config) error {
+	cf, err := loadConfig(ctx)
+	if err != nil {
+		return err
+	}
+
+	if cf.FocusedServer().KubeConfigs == nil {
+		cf.FocusedServer().KubeConfigs = make(map[string]*api.Config)
+	}
+	cf.FocusedServer().KubeConfigs[fmt.Sprintf(kubeConfigKeyFormat, user, cf.FocusedServer().FocusedCluster())] = kubeConfig
```

Comment:
nit: should `cf.FocusedServer()` be just one call that gets stored in a variable?

Reply:
It's just access a map so it's not an expensive call, but it would probably be more readable, will change.

Reply:
Using a variable doesn't work for L174 since I want to assign a new values to one of it's fields.